server/api/google_login_handler.py

```diff
@@ -20,6 +20,15 @@ def googlelogin():
             user_db = Users.query.filter_by(google_id=userid).first()
             uid = user_db.id
             token = token_generator(uid)
-            return jsonify({'response': 'Login success', 'token': token, 'id': uid}), 200
+            
+            ret = jsonify({'response': 'Login success', 
+                            'id': uid,
+                            'user_url': user_db.url,
+                            'user_timezone': user_db.timezone,
+                            'user_available_day': user_db.available_day,
+                            'user_available_time': user_db.available_time
+                            })
+            ret.set_cookie("token", token, httponly = True)
```

Comment:
@bugatti12345 cookie creation was moved to server side because we can't set HTTP_only cookie on the frontend. I found out this after trying to get cookie from frontend but failed. 

Here is more info:
https://stackoverflow.com/questions/14691654/set-a-cookie-to-httponly-via-javascript